src/test/java/org/zalando/nakadi/controller/EventTypeControllerTest.java

```diff
@@ -472,6 +473,43 @@ public void whenDeleteEventTypeThen403() throws Exception {
     }
 
     @Test
+    public void whenDeleteEventTypeNotAdminAndDeletionDeactivatedThenForbidden() throws Exception {
```

Comment:
These two methods are almost the same except the very last line. 
Maybe you can extract a helper method here.